docs/legate/core/source/api/index.rst

```diff
@@ -0,0 +1,47 @@
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+API Reference
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. currentmodule:: legate.core
+
+.. toctree::
+   :maxdepth: 1
+
+.. autosummary::
+   :toctree: generated/
+
+    Point
```

Comment:
Note that the classes listed here aren't necessarily what we want to expose to the developers, as they are Legion abstractions and not Legate's. I'm fine with merging this PR, but keep in mind that we will need to revisit this API reference in the future.

Reply:
:+1:  For sure, just duplicates the pre-existing structure and gets it back to successfully building.